ARMTemplates/AzFunctionApp-Consumption.JSON

```diff
@@ -0,0 +1,141 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+	  "appName": {
+		"type": "string",
+		"defaultValue": "[concat('fnapp', uniqueString(resourceGroup().id))]",
+		"metadata": {
+		  "description": "The name of the function app that you wish to create."
+		}
+	  },
+	  "storageAccountType": {
+		"type": "string",
+		"defaultValue": "Standard_LRS",
+		"allowedValues": [
+		  "Standard_LRS",
+		  "Standard_GRS",
+		  "Standard_RAGRS"
+		],
+		"metadata": {
+		  "description": "Storage Account type"
+		}
+	  },
+	  "location": {
+		"type": "string",
+		"defaultValue": "[resourceGroup().location]",
+		"metadata": {
+		  "description": "Location for all resources."
+		}
+	  },
+	  "runtime": {
+		"type": "string",
+		"defaultValue": "node",
```

Comment:
This should default to `dotnet`, not `node`. `dotnet` should be the only allowed value.

Reply:
Yes i agree....typo looks like....will fix